### PR TITLE
Describe this repository

### LINKED ISSUE #1: This repository has no description
This repository should describe itself and explain its purpose, because GitHub Actions is new to the business unit AWS and, I believe, the evoila Group at large.

In particular, this repository should have a description (the body parameter `description` in the [API method to update repositories](https://docs.github.com/en/rest/repos/repos#update-a-repository)) and a [README](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes).

### PR DESCRIPTION
**Pull request by kalrish**
I propose that both this repository itself as well as its directories for [reusable workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows) (`.github/workflows`) and [custom actions](https://docs.github.com/en/actions/creating-actions/about-custom-actions) (`.github/actions`) be described in [READMEs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes).

Furthermore, I propose that we pin actions that we call but are not maintained by

- ourselves (whether in this or another repository),
- [GitHub itself](https://github.com/actions) or
- any individuals and organizations we trust,

as [recommended by GitHub](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions):

> Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps mitigate the risk of a bad actor adding a backdoor to the action's repository, as they would need to generate a SHA-1 collision for a valid Git object payload.

Merging this pull request would resolve #1.

### CODE REVIEW

**Comment by kalrish** [comment]
The following commands should run without errors, @EduardSchwarzkopf:

```
$ npm ci
$ npx -- markdownlint-cli2
```

I have only tested on Linux, though.

The [subcommand `ci`](https://docs.npmjs.com/cli/v9/commands/npm-ci) is like [`install`](https://docs.npmjs.com/cli/v9/commands/npm-install) but additionally ensures the lock file is working.

**Review by EduardSchwarzkopf — DISMISSED** [review]
Looking good, the test was also ok. I've tested also a wrong formated .md file and it showed me errors.

File: .github/workflows/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Most of them are [reusable](https://docs.github.com/en/actions/using-workflows/reusing-workflows).
+
+Workflows meant only for this repository and reusable workflows meant for the organization should be in separate directories.
+Unfortunately, all workflows must be located in this directory.
+This limitation has been brought up in at least two GitHub public feedback discussions:
+
+- [#9050](https://github.com/orgs/community/discussions/9050) and
+- [#10773](https://github.com/orgs/community/discussions/10773).
+
+Subdirectories, which would help keep reusable workflows tidy, are not allowed either.
```
Comment (EduardSchwarzkopf) [inline .github/workflows/README.md:13]: Are subdirectories not allowed because of GitHub itself?

**Review by EduardSchwarzkopf — APPROVED** [review]
I just this one question regarding the subdirectories on github actions, but should not holding the merge

File: .github/workflows/README.md (hunk 1 above)
Comment (kalrish) [reply .github/workflows/README.md:13]: Yes, GitHub itself disallows both subdirectories and alternative directories, unfortunately. Should I rephrase that README to make it more clear?

**Pull request merged**